lib/getStagedFiles.js

```diff
@@ -4,26 +4,11 @@ import normalize from 'normalize-path'
 
 import { execGit } from './execGit.js'
 import { parseGitZOutput } from './parseGitZOutput.js'
+import { getDiffCommand } from './getDiffCommand'
```

Comment:
This import needs to include the `.js` extension

Reply:
Also, if you don't mind, can you sort these alphabetically by the file name:

```suggestion:-2+3
import { execGit } from './execGit.js'
import { getDiffCommand } from './getDiffCommand'
import { parseGitZOutput } from './parseGitZOutput.js'
```